TAScheduler/AcceptanceTests/test_login.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase, Client
+from TAScheduler.models import User
+
+class SuccessfulLogin(TestCase):
+    def setUp(self):
+        self.client = Client()
+        User.objects.create_user(email_address="user@example.com", password="password")
```

Comment:
User should throw an error if you create a user without all files filled so you need to do this for all your tests.

Reply:
Isn't that more of Unit test issue?

Reply:
I think Kevine was talking about the "error message" not necessarily an "exception" as seen on the prototypes

Reply:
I understand, I'm just confused as to why that would be an acceptance test and not a Unit test. If we agree they should be there I can still add them.

Reply:
Oh I think he might have been saying that, in creating a User, if you do not provide all the inputs it will throw an error. I am not certain if that's what he meant and why this would happen - if it's true.

(Sorry if I am misunderstanding your concern - it'll be best to followup with Kevin on what he meant if this still doesn't answer your question)

Reply:
I am fairly sure if you try to create a user without all fields Django throws an error because we do not have default values defined for all fields. So you need to fill all values when creating an object for the databae

Reply:
Ohhhh, I see what you're saying. As of now, I'm using Django's premade User function. What is the right way to import user from models? I was going to try to change it, but got stuck on that.

Reply:
nevermind, i had a brainfart